### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by ChunmianLin** [Byronnar/tensorflow-serving-yolov3#10]
你好，你看得懂他代码中关于conf_loss计算的部分吗？那个focal是表示什么呢，然后下面对含目标和对不含目标的的损失计算，都采用了sigmoid CE 的方法，且输入的都一样，这样怎么区分是背景还是目标的loss贡献呢？

**Comment by Byronnar** [comment]
你好，关于这个损失函数的细节我也还没有仔细研究哎，这个可以问问原作者。https://github.com/YunYang1994/tensorflow-yolov3
